src/content/terraform/product-landing.json

```diff
@@ -51,58 +51,56 @@
 	"blocks": [
 		{
 			"type": "heading",
-			"heading": "Use Cases",
+			"heading": "Featured Documentation",
 			"level": 2,
 			"size": 400
 		},
-		{
-			"type": "tutorial_cards",
-			"tutorialSlugs": ["vault/transform", "vault/static-secrets"]
-		},
 		{
 			"type": "linked_cards",
 			"cards": [
 				{
-					"heading": "Dolor sit amet",
-					"body": "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incident this text is written so that it's very very long but it should be truncated in the card to a max of three lines to match designs",
-					"url": "https://www.terraform.io"
+					"heading": "Configuration Language",
+					"body": "Describe infrastructure in Terraform configuration language.",
```

Comment:
Do we want to specifically call out HCL in the body?

Reply:
we thought about it but you can also write terraform configuration in JSON, not that we have much documentation on it, but I'd like to leave the door open for that in the future?